installing/installing_openstack/installing-openstack-installer-custom.adoc

```diff
@@ -18,17 +18,6 @@ processes.
 
 * Have metadata service enabled in {rh-openstack}
 
-[IMPORTANT]
-====
-{product-title} 4.3 is supported for use with {rh-openstack} 13 and
-{rh-openstack} 16.
-
-The latest {product-title} release supports both the latest {rh-openstack} long
```

Comment:
There's more data in this note than in the arch guide, so I hesitate to just remove it. I suggest implementing one of these changes:

A. Update the arch guide to contain more nuance about the type of OSP releases that you're supporting. Amend the prereq about consulting the arch guide to point out that you need to care about the versions.

B. Make a module about supported OSP versions and include it in the assemblies, like [vSphere](https://docs.openshift.com/container-platform/4.2/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-infrastructure_installing-vsphere). Pull the details of supported OSP versions from the arch doc.

As long as the details are in only one place, I'm fine with either. I'd probably do B and swing it by Eric Rich.